examples/api/test/material/app_bar/sliver_app_bar.3_test.dart

```diff
@@ -29,15 +27,21 @@ void main() {
     final BuildContext context = tester.element(find.byType(MaterialApp));
     final TextStyle expectedTitleStyle = Theme.of(context).textTheme.headlineMedium!;
 
-    // There are two Text() widgets: expanded and collapsed. The expanded is first.
+    // There are two Text widgets: expanded and collapsed. The expanded is first.
     final Finder titleFinder = find.text(title).first;
     final TextStyle actualTitleStyle = DefaultTextStyle.of(tester.element(titleFinder)).style;
 
     expect(actualTitleStyle, expectedTitleStyle);
 
-    expect(tester.getBottomLeft(find.text(title).first).dy, 124.0);
-    await tester.drag(find.text(title).first, _kOffset, touchSlopY: 0, warnIfMissed: false);
+    // Scrolling the screen moves the title up.
+    expect(tester.getBottomLeft(titleFinder).dy, 124.0);
+    await tester.drag(
+      titleFinder,
+      const Offset(0.0, -200.0),
+      touchSlopY: 0,
+      warnIfMissed: false);
```

Comment:
```suggestion
      warnIfMissed: false,
    );
```

(nit, per the [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation))

Reply:
Of course! And thanks to you for being receptive to feedback and taking the time to contribute ❤️